client/blocks/comment-detail/index.jsx

```diff
@@ -82,16 +82,31 @@ export class CommentDetail extends Component {
 
 	toggleApprove = () => {
 		const { commentId, commentStatus, postId, setCommentStatus } = this.props;
-		setCommentStatus( commentId, postId, 'approved' === commentStatus ? 'unapproved' : 'approved' );
+		const isPersisted = 'approved' === commentStatus || 'unapproved' === commentStatus;
+
+		setCommentStatus( commentId, postId, 'approved' === commentStatus ? 'unapproved' : 'approved', {
```

Comment:
This is a leftover from the Faker days, where we only used the `commentId`.
Real actions though also require `siteId` and `postId`.
`siteId` is passed inside `mapDispatchToProps` for maximum convenience, therefore we're left with `postId` and `commentId`.

I know that our usual argument order is "from larger to smaller context" (`siteId, postId, commentId`), but in this case using `postId` before `commentId` feels a bit weird to me.

Though, should we want to modify this, I'd do it in another PR to avoid cluttering this one with unrelated changes.